src/sass/table/table.scss

```diff
@@ -64,6 +68,34 @@ $density-relaxed-img-radius: 8px;
     padding: $density-relaxed-padding;
 }
 
+.table--frozen-header thead {
```

Comment:
I might not have full context, but I am see everywhere else using `sticky` instead of `frozen` even in figma playbook.

Reply:
We aren't necessarily bound to the vernacular of DS. We chart our own course for what makes sense for the web/users. Often times, DS actually reevaluates their terminology and follows suit as well.

I used `frozen` instead of `sticky` since that is par for the course on user-level understanding of table cells. We can rename if needed, but I'm not seeing the naming as an issue.